frontend/src/css/IconsRtl.styl

```diff
@@ -1,14 +1,3 @@
 
-[dir=rtl] .fa-hand-point-right
-[dir=rtl] .fa-caret-right
-[dir=rtl] .fa-chevron-right
-[dir=rtl] .fa-chevron-left
-[dir=rtl] .fa-share-alt
-[dir=rtl] .fa-sign-out-alt
-[dir=rtl] .fa-cogs
-[dir=rtl] .fa-tag
-[dir=rtl] .fa-magic
-[dir=rtl] .fa-folder-open
-[dir=rtl] .fa-folder
-[dir=rtl] .fa-comments
-	transform scale(-1,1)
+[dir=rtl] i
```

Comment:
You need to update the issue if you are not gonna use .fa

Reply:
done, thanks :)